### PR TITLE
KILL ancient macros

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request
Removes TAN and CLAMP Macros

## Why It's Good For The Game
they were built into byond math YEARS ago. Yeet these macros!

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl: rkz
code: Death upon ye, ancient math!
/:cl:

### CODE REVIEW

File: code/modules/library/lib_machines.dm [hunk 1]
```diff
@@ -304,7 +304,7 @@ GLOBAL_LIST(cachedbooks) // List of our cached book datums
 				dat += "<A href='?src=[REF(src)];orderbyid=1'>(Order book by SS<sup>13</sup>BN)</A><BR><BR>"
 				dat += "<table>"
 				dat += "<tr><td>AUTHOR</td><td>TITLE</td><td>CATEGORY</td><td></td></tr>"
-				dat += libcomp_menu[CLAMP(page,1,libcomp_menu.len)]
+				dat += libcomp_menu[clamp(page,1,libcomp_menu.len)]
```
Comment (itsmeow) [inline code/modules/library/lib_machines.dm:307]: uhhmmmmm uh um what?!? syntax??

Comment (Tsar-Salat) [reply code/modules/library/lib_machines.dm:307]: clamped

**Review by PowerfulBacon — APPROVED** [review]
Yea there is just no reason for these to exist really. I would argue against consistency, however sin and cos are already in lowercase so it makes sense that tan would be too. If anything the argument for consistency would be against these macros and for making them lower case proc-like macros

File: code/modules/library/lib_machines.dm (hunk 1 above)
Comment (itsmeow) [reply code/modules/library/lib_machines.dm:307]: don't code review while distracted guys you forget how array index operators work

**Pull request merged**